openvino_xai/metrics/adcc.py

```diff
@@ -77,6 +77,16 @@ def complexity(saliency_map: np.ndarray) -> float:
         """
         return abs(saliency_map).sum() / (saliency_map.shape[-1] * saliency_map.shape[-2])
 
+    def _scale_map(self, saliency_map: np.ndarray) -> np.ndarray:
+        """
+        Scale the saliency map to [0, 1] range.
+        """
+        if np.min(saliency_map) < 0:
+            saliency_map = scaling(saliency_map, cast_to_uint8=False, max_value=1)
+        elif np.max(saliency_map) > 1:
+            saliency_map = saliency_map / 255
+        return saliency_map
```

Comment:
I wanted to make a small optimization, since in most cases we'll get the second option where the saliency map is already scaled [0,255]. In this case, I simply divide by 255 to make it [0,1] instead of calculating min and max again and actual scaling. But I guess, this overhead is nothing, so aI don't need to make an extra condition